scripts/delete-sourcify-match.ts

```diff
@@ -677,8 +699,19 @@ async function deleteVerifiedContracts(
       `✓ Deleted ${stats.codeEntriesDeleted} code entrie(s) (not referenced elsewhere)`,
     );
 
-    await client.commitTransaction();
-    console.log("\n✅ Transaction committed successfully\n");
+    // Check if this is a dry run
+    const isDryRun = process.env.DRY_RUN === "true";
+
+    if (isDryRun) {
+      await client.rollbackTransaction();
```

Comment:
what's the need for dryrun if you rollback in it? just to see if it runs? I see this useful if we can somehow output what it would be deleted

Reply:
Ah sorry I didn't notice!